src/components/AuthoritiesSearchForm/AuthoritiesSearchForm.js

```diff
@@ -118,40 +110,40 @@ const AuthoritiesSearchForm = ({
   const isResetAllButtonDisabled = (!searchInputValue && !isFiltersApplied) || isAuthoritiesLoading;
 
   return (
-    <form onSubmit={onSubmitSearch}>
-      <FilterNavigation />
-      <div className={css.searchGroupWrap}>
-        <SearchTextareaField
-          textAreaRef={searchInputRef}
-          autoFocus
-          rows="1"
-          name="query"
-          id="textarea-authorities-search"
-          className={css.searchField}
-          searchableIndexes={searchableIndexes}
-          onSubmitSearch={onSubmitSearch}
-        />
-        <Button
-          id="submit-authorities-search"
-          data-testid="submit-authorities-search"
-          type="submit"
-          buttonStyle="primary"
-          fullWidth
-          marginBottom0
-          disabled={isSearchButtonDisabled}
-        >
-          {intl.formatMessage({ id: 'ui-marc-authorities.label.search' })}
-        </Button>
-      </div>
-      <AdvancedSearch
-        open={isAdvancedSearchOpen}
-        searchOptions={advancedSearchOptions}
-        defaultSearchOptionValue={searchableIndexesValues.KEYWORD}
-        firstRowInitialSearch={advancedSearchDefaultSearch}
-        onSearch={handleAdvancedSearch}
-        onCancel={() => setIsAdvancedSearchOpen(false)}
-      >
-        {({ resetRows }) => (
+    <AdvancedSearch
```

Comment:
There is a problem, when you wrap the form into `AdvancedSearch` component. [AdvancedSearch](https://github.com/folio-org/stripes-components/blob/master/lib/AdvancedSearch/AdvancedSearch.js#L127) uses `enter` handler to apply advanced search. So this will cause problems if you submit form by hitting `enter`: you are calling `handleAdvancedSearch` when you don't to, and the indexes and search value get confused as a result.

Reply:
Hi, @ usavkov-epam 
I wrap the form into AdvancedSearch component, because
rowState is available in this component and we need to pass in onSubmitSearch in form. I see that it is a problem, when submit form by hitting enter, but cann't fixed it. I'll try look for more ways to do it.